jina/clients/python/request.py

```diff
@@ -98,7 +99,7 @@ def _generate(data: Union[Iterator['jina_pb2.Document'], Iterator[bytes], Iterat
                 req.queryset.extend([top_k_queryset])
 
         for content in batch:
-            _add_document(request=req, content=content, mode=mode, doc_counter=doc_counter,
+            _add_document(request=req, content=content, mode=mode, doc_counter=doc_counter if override_doc_id else None,
```

Comment:
That just became impossible to grasp when just skimming the function. Could you put one parameter per line in order to make it more obvious, that something else then just parameter passing is happening here?